howso/upgrade.amlg

```diff
@@ -198,12 +198,16 @@
 			;base path to Howso Engine Core installation
 			root_filepath (null)
 			;{type "string"}
+			;path from which to load previously exported amlg trainee
+			;	If specified, trainee_json_filepath is ignored
+			trainee_amlg_filepath (null)
+			;{type "string"}
 			;path from which to load previously exported meta.json and exp.json files.
 			;	If unspecified, expects them to be located in the base installation /migrations/ directory.
 			trainee_json_filepath (null)
 			;{type "boolean"}
-			;flag, if true will load each case from its individual file
-			separate_files (false)
+			;if true, then will import and use the trainee information in the subtrainee named ".old_trainee"
```

Comment:
why does user need to know about a named subtrainee for the flow, this comment should be clearer, ie flag used by subtrainees when upgrading themselves automatically.  perhaps even rename the flag to match?

Reply:
well users could use this if they for some reason wanted to load the trainee themself